sklearn/linear_model/tests/test_quantile.py

```diff
@@ -28,6 +28,10 @@ def default_solver():
     return "highs" if sp_version >= parse_version("1.6.0") else "interior-point"
 
 
+@pytest.mark.skipif(
+    sp_version > parse_version("1.10.999"),
```

Comment:
Nit: This looks like the following:

```suggestion
    sp_version >= parse_version("1.11"),
```

(Same for the other changes.)

Reply:
This would not work but let me propose:

```
parse_version(sp_version.base_version) >= parse_version("1.11")
```

The issue here is that we don't want it to fail for the pre-release version.